pkg/openthread/contrib/netdev2/openthread_netdev2.c

```diff
@@ -0,0 +1,191 @@
+/*
```

Comment:
This file should be renamed in `openthread_netdev.c` and parent directory should be `netdev`. There might be other cases like this.